main/templates/main/about.html

```diff
@@ -0,0 +1,16 @@
+{% extends "main/base.html" %}
+{% block content %}
+    <div class="container mt-5">
+        <div class="row">
+            <div class="col-md-12">
+                <h1>About the Project</h1>
+                <p>
+                    The Digital Research Competencies Framework is designed to help researchers identify the skills they need, define development pathways, and progress their careers in research software.
+                </p>
+                <p>
+                    For more information, please contact us at <a href="mailto:">email</a>.
```

Comment:
We should pass the value for this email in as context data from the view. So it can be:

```suggestion
                    For more information, please contact us at <a href="mailto:">{{ email }}</a>.
```

We currently don't have a contact email defined, but we should probably define it in settings. That isn't to be done in this PR, but should be included at some point. So for now just pass in the string "email" and mark it as a todo

Reply:
@AdrianDAlessandro @davehorsfall I have now removed the text that references to the email from the html.